src/components/Blinds/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import "./style.css"
+import { useState } from "react"
+
+import blindsOpen from "./blinds-open.svg"
+import blindsClosed from "./blinds-closed.svg"
+
+export const Blinds = ({ state }) => {
+    const [stateBlinds, setStateBlinds] = useState(state)
+
+    const open = () => {
+        setStateBlinds('open')
+    }
+
+    const closed = () => {
+        setStateBlinds('closed')
+    }
+    
+    // if open click on this, else close
+    // const handleClick = () => {
+    //     if (stateBlinds === "open") {
+    //         setStateBlinds = ("blindsOpen")
+    //     } else {
+    //         setStateBlinds = ("blindsClosed")
+    //     }
+    // }
```

Comment:
tohle by taky šlo, nebo i 
```
     const handleClick = () => {
             setStateBlinds(stateBlinds ? "blindsOpen" : "blindsClosed")
     }
```

Reply:
akorát bacha na to že "setStateBlinds" je funkce, tj jí voláš (`setStateBlinds(xxx)`). Není to proměnná do které bys přiřadila hodnotu (setStateBlinds = xxx)